example/server/ui.ts

```diff
@@ -175,6 +175,41 @@ function getVideoControls(mediaPath: string, state: VideoMedia, context: UIConte
         options: { $: 'ref', ref: ['videoList'] },
       },
     },
+    {
+      $: 'component',
+      component: 'RiseForm',
+      props: {
+        onSubmit: {
+          $: 'event',
+        },
+      },
+      children: [
+        {
+          $: 'component',
+          component: 'RiseTextField',
+          props: {
```

Comment:
This form is obviously in the wrong place and with wrong values, but just wanted to give you an example. Let me know where this should be rendered and will update accordingly.

![Uploading Screenshot 2024-04-20 at 13.45.37.png…]()

Reply:
Update from the discussion: We're going to build title editor for layers/sequences.